src/app/options.h

```diff
@@ -1,5 +1,7 @@
 /*
- * Bittorrent Client using Qt4 and libtorrent.
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2016  Eugene Shalygin
```

Comment:
I don't know if this intended or not. This is just a reminder. You might want to leave an email beside your name.
If yes, add it on every other place you put your name.